### PR TITLE
Update README.md to add supported functions

### PR DESCRIPTION
**Pull request by logankilpatrick**

### CODE REVIEW

**Review by schnerd — APPROVED** [review]
Left some feedback in Slack, but pre-approving.

File: README.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ yarn add openai
 
 ## Usage
 
-> [!IMPORTANT]
-> Previous versions of this SDK used a `Configuration` class. See the [v3 to v4 migration guide](https://github.com/openai/openai-node/discussions/217).
+The full API of this library can be found in [API.md file](https://github.com/openai/openai-node/blob/master/api.md). The code below shows how to get started using the chat completions API.
```
Comment (logankilpatrick) [inline README.md:21]:
```suggestion
The full API of this library can be found in [API.md file](https://github.com/openai/openai-node/blob/master/api.md). The code below shows how to get started using the Chat completions API.
```

Comment (logankilpatrick) [reply README.md:21]: I think you said you wanted this convention right @schnerd ?

Comment (schnerd) [reply README.md:21]: Gray area, but based on documentation style guide i'm leaning towards keeping this lowercase.

File: README.md [hunk 2]
```diff
@@ -18,8 +18,7 @@ yarn add openai
 
 ## Usage
 
-> [!IMPORTANT]
-> Previous versions of this SDK used a `Configuration` class. See the [v3 to v4 migration guide](https://github.com/openai/openai-node/discussions/217).
+The full API of this library can be found in [API.md file](https://github.com/openai/openai-node/blob/master/api.md). The code below shows how to get started using the Chat completions API.
```
Comment (logankilpatrick) [inline README.md:21]:
```suggestion
The full API of this library can be found in [API.md file](https://github.com/openai/openai-node/blob/master/api.md). The code below shows how to get started using the chat completions API.
```

**Comment by rattrayalex** [comment]
LGTM too! We'll incorporate this soon.

**Review by schnerd — APPROVED** [review]

**Pull request merged**